src/safe-upgrade-tool/install.md

```diff
@@ -0,0 +1,100 @@
+---
+group: software-update-guide
+title: Install
+ee_only: True
+functional_areas:
+  - Upgrade
+---
+
+The Safe Upgrade Tool ALPHA (SUT) is a command line (CLI) tool that checks a Magento instance against a specific version by analyzing all the non-Magento modules installed on it.
+
+{:.bs-callout-warning}
+At the moment this is an ALPHA version with limited scope, only validating PHP Magento APIs and GraphQL schema.
+
+## Workflow
+
+The following diagram shows the expected workflow when running the SUT:
+
+![SUT Diagram](img/mvp-diagram.png)
+
+### Who is the SUT for?
+
```

Comment:
Let's add this here:
"The SUT Alpha is now available for all Magento Commerce merchants, and it can be downloaded on Magento repository [repo link]."

### SUT main use case

Reply:
added link